src/widget/whotcuebutton.cpp

```diff
@@ -112,6 +118,12 @@ void WHotcueButton::setup(const QDomNode& node, const SkinContext& context) {
     }
 }
 
+bool WHotcueButton::isActivate() const {
```

Comment:
```suggestion
bool WHotcueButton::isActive() const {
```
Please also rename to `m_pCoActive`, `READ isActive` etc.